modin/tests/pandas/dataframe/test_map_metadata.py

```diff
@@ -223,13 +223,51 @@ def test_add_suffix(data, axis):
     df_equals(new_modin_df.columns, new_pandas_df.columns)
 
 
+map_strtagies = ["map", "axis_map", "splitted_axis_map"]
```

Comment:
```suggestion
map_strategies = ["map", "axis_map", "splitted_axis_map"]
```

Reply:
Outdated since the test was moved to test_internals.